myhoard/basebackup_operation.py

```diff
@@ -21,6 +21,14 @@
 CURSOR_TIMEOUT_DURING_OPTIMIZE: int = 120
 
 
+class AbortRequested(Exception):
+    """The base backup operation was aborted by request."""
+
+    def __init__(self, reason):
+        super().__init__()
```

Comment:
The exception would be more informative in logs and traces if a message is passed to the base:
```suggestion
        super().__init__(f"reason: {abort_reason}")
```

This turns `print(error)` into `AbortRequested("reason: something")` instead of the mysterious `AbortRequested()`.